Assets/scss/base/_var.scss

```diff
@@ -91,6 +102,13 @@ $font-normal:normal;
     --card-bg-color: var(--white);
     --card--title-color: var(--darkestgrey);
 
+    --profile-card-light-background-color: var(--magenta-pink);
+    --profile-card-dark-background-color: var(--dark-red);
+    --profile-card-background-color: var(--white);
+    --profile-card-hover-background-color: var(--kingly-cloud);
+    --profile-card-hover-light-background-color: var(--dark-orange);
+    --profile-card-hover-dark-background-color: var(--burlywood);
```

Comment:
Could scope these props within `.cmp-profile-card` since that's where they are used, while `_var.scss`. Not mandatory since these are named with a convention which obviously applies to only that component, just another way to organize things potentially.